core/js/typespecs/openai/messages.ts

```diff
@@ -79,7 +79,10 @@ const functionSchema = z.object({
   name: z.string(),
 });
 const chatCompletionToolMessageParamSchema = z.object({
-  content: z.string().default(""),
+  content: z.union([
```

Comment:
To make stainless happy, all unions must have a title property (see other unions in this file).